victorialogs: marked fluentd support in roadmap, added syslog example (#7098)
VictoriaMetrics/VictoriaMetrics: deployment/docker/victorialogs/fluentd/README.md

# Docker compose Fluentd integration with VictoriaLogs

The folder contains examples of [Fluentd](https://www.fluentd.org/) integration with VictoriaLogs using protocols:

* [loki](./loki)
* [jsonline](./jsonline)
* [elasticsearch](./elasticsearch)

> All required plugins, that should be installed in order to support protocols listed above can be found in a [Dockerfile](./Dockerfile)

Reviewer: ```suggestion
All required plugins for protocols listed above can be found in a [Dockerfile](./Dockerfile)
```

The docker compose file contains the following components:

> * fluentd - Fluentd logs collection agent, which is configured to collect and write data to `victorialogs`

Reviewer: ```suggestion
* fluentd - logs collection agent, configured to collect and write data to `victorialogs`
```

> * victorialogs - VictoriaLogs log database, which accepts the data from `fluentd`

Reviewer: ```suggestion
* victorialogs - logs database, receives data from `fluentd` agent
```

> * victoriametrics - VictoriaMetrics metrics database, which collects metrics from `victorialogs` and `fluentd`

Reviewer: ```suggestion
* victoriametrics - metrics database, collects metrics from `victorialogs` and `fluentd` for observability purposes
```
